### PR TITLE
fix: french loading tips

### PR DESCRIPTION
**Pull request by Rapx3**
Fixes #[issue_no]

## Summary of Changes

* Fix punctuation spaces.
* Fix units case + typo
* Added french translation for  "COCKPIT.TOOLTIPS.APU_AVAILABLE" String.

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: A32NX/fr-FR.locPak [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
+      "A32NX.LOADING.TIPS.A320NEO_002": "Le saviez-vous ? L'A320 peut atterrir par vent de travers jusqu'à 38 nœuds et avec un vent arrière jusqu'à 15 nœuds !",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire tire son nom des commandes de vol électriques introduites pour la première fois dans l'Airbus A320.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "Il y a 9693 avions de la famille A320 construits! C'est presque autant que la population du Tuvalu!",
-      "A32NX.LOADING.TIPS.A320NEO_005": "Les avions Airbus A320neo ont une autonomie de 3500 NM ou 6500 KM. C'est suffisant pour voler de New York à Paris!",
-      "A32NX.LOADING.TIPS.A320NEO_006": "Le saviez-vous? La famille A320 a 4 variantes! L'A318, l'A319, l'A320 et l'A321."
+      "A32NX.LOADING.TIPS.A320NEO_004": "Il y a 9693 avions de la famille A320 construits ! C'est presque autant que la population des Tuvalu !",
+      "A32NX.LOADING.TIPS.A320NEO_005": "Les avions Airbus A320neo ont une autonomie de 3500 NM ou 6500 km. C'est suffisant pour voler de New York à Paris !",
```
Comment (MisterChocker) [inline A32NX/fr-FR.locPak:16]: what about the nm? do you guys write km lower case but nm upper case?

Comment (Rapx3) [reply A32NX/fr-FR.locPak:16]: Good question ! I doubled check on [Wikipedia](https://fr.wikipedia.org/wiki/Mille_marin) but also this [paper](https://www.nist.gov/system/files/documents/pml/div684/fcdc/si_brochure_8.pdf), from International Bureau of Measurements (approximate translation) : check the page 42, table 8, (d). There is not international agreement on the unit, but it’s commonly written NM.

Comment (MisterChocker) [reply A32NX/fr-FR.locPak:16]: I looked it up myself and the upper case NM is actually the agreement I found in US and Germany.

image

Comment (beheh) [reply A32NX/fr-FR.locPak:16]: There is actually [guidance on this](https://aerosavvy.com/wp-content/uploads/2014/08/an05_cons.pdf) from ICAO: 

image

Specifically the SI unit km is lowercase, but the alternative NM is uppercase.

File: A32NX/fr-FR.locPak [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
+      "A32NX.LOADING.TIPS.A320NEO_006": "Le saviez-vous ? La famille A320 a 4 variantes! L'A318, l'A319, l'A320 et l'A321."
```
Comment (Benjozork) [inline A32NX/fr-FR.locPak:17]:
```suggestion
      "A32NX.LOADING.TIPS.A320NEO_006": "Le saviez-vous ? La famille A320 a 4 variantes ! L'A318, l'A319, l'A320 et l'A321."
```

**Comment by Benjozork** [comment]
@Rapx3 This needs to be fixed ASAP to be in time for v0.6.0

**Review by BillyGeoThink2020 — APPROVED** [review]

**Comment by Rapx3** [comment]
@Benjozork , it should be fixed.

**Review by Benjozork — APPROVED** [review]
LGTM. Merci beaucoup.

**Comment by Benjozork** [comment]
Can you provide your Discord username so I can give you the Contributor role?

**Comment by Rapx3** [comment]
Il n’y a pas de quoi !
Here’s my discord username : Rapx3#6301

**Pull request merged**